scripts/accumulator.key

```diff
@@ -0,0 +1,28 @@
+-----BEGIN PRIVATE KEY-----
```

Comment:
As discussed by skype, this file will be generated on-the-fly during test harness, so it needs to be removed from repo.